pkg/attestation/crafter/runners/githubaction.go

```diff
@@ -22,33 +22,37 @@ import (
 
 	schemaapi "github.com/chainloop-dev/chainloop/app/controlplane/api/workflowcontract/v1"
 	"github.com/chainloop-dev/chainloop/pkg/attestation/crafter/runners/oidc"
+	"github.com/rs/zerolog"
 )
 
 type GitHubAction struct {
 	githubToken *oidc.Token
 }
 
-func NewGithubAction(ctx context.Context) *GitHubAction {
+func NewGithubAction(ctx context.Context, logger *zerolog.Logger) *GitHubAction {
```

Comment:
what about logging the actual code of this module?

I'd store the logger in the `GitHubAction` struct and then log with debug the process of retrieving the token, verifying it, etc. What do you think?